tasks/task4.js

```diff
@@ -0,0 +1,58 @@
+/* Task 1 */
+function createObject(obj) {
+  const obj1 = obj;
```

Comment:
Make function universal.
Not only for getTestField method

Reply:
completed